web/app/themes/clarity/inc/admin/acf-field-group.php

```diff
@@ -5904,37 +5879,10 @@
 			'ui_on_text' => '',
 			'ui_off_text' => '',
 		),
-		array(
```

Comment:
Why is agency_wide_banner_type being removed?

Reply:
Yes I noticed this and the OSPT fields being removed, it's like ACF reverted back to some previous state. I'm not sure how that is possible though, given it is a PHP file. I'm having a look now, thanks